proto-public/pbmesh/v1alpha1/proxy_configuration.proto

```diff
@@ -29,34 +30,72 @@ message ProxyConfiguration {
   google.protobuf.Struct opaque_config = 4 [deprecated = true];
 }
 
+// This is a Resource type. Tenancy scoping applies.
+message WildcardProxyConfiguration {
```

Comment:
I thought we said we'll call them `Partition...` based on feedback from PMs

Reply:
Meant to remove from this PR .